src/components/Form/Form.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Input from '../Input/Input';
+
+function Form(props) {
+  const { title, inputArr, tertiaryButton } = props;
+  const [isValid, setIsValid] = useState('');
+  const [amountTrueArr, setAmountTrueArr] = useState([]);
+
+  const onSubmit = (e) => {
```

Comment:
avoid abbreviations in code. The extra characters are generally worth the readability